src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.reader.CsvFileReader;
+import core.basesyntax.report.CsvFileWriter;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReader csvFileReader = new CsvFileReader("testData1.csv");
+        Map<String, Integer> fruitQuantities = csvFileReader.readTransactions();
+
+        CsvFileWriter csvFileWriter = new CsvFileWriter("report.csv");
+        csvFileWriter.writeReport(fruitQuantities);
```

Comment:
Make your services independent and call them in the `main()` method. The `CsvFileWriter` should be called with the processed data, not directly with the raw quantities map.